### PR TITLE
Initial project skeleton for icalendar

### PR DESCRIPTION
**Pull request by ennamarie19**
This is my first contribution to OSS-Fuzz! I would greatly appreciate any feedback and confirmation that I have configured my project.yaml correctly. 

Thank you for your time and review!

### CODE REVIEW

**Comment by ennamarie19** [comment]
I believe this project is important because it is used for parsing and generating iCalendar files following the standards in RFC 2445 and RFC 5545. This repository answers the need outlined in RFC2245 of "providing and deploying interoperable calendaring and scheduling services for the Internet." Instead of having to create ad-hoc tools to parse and generate iCalendar files, this Python repository provides an encompassing yet simple means to do so. In other words, icalendar allows for interoperability between calendar services and protocols. With nearly 900 stars on GitHub, this package has a wide user base and has spawned other popular projects such as icalevents.

File: projects/icalendar/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-python
+RUN apt-get update && apt-get install -y make autoconf automake libtool
+RUN git clone --depth 1 <git_url> icalendar     # or use other version control
```
Comment (jonathanmetzman) [inline projects/icalendar/Dockerfile:19]: Oh please fill this part in.

File: projects/icalendar/build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
```
Comment (jonathanmetzman) [inline projects/icalendar/build.sh:18]: If you don't want to actually write code at this moment, just delete the build.sh and Dockerfile and leave the project.yaml, but if we do commit them, they have to work.

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]

**Comment by jonathanmetzman** [comment]
We approve of adding this project. The PR needs a little work though

**Comment by ennamarie19** [comment]
@jonathanmetzman I have just finalized my initial integration. I would greatly appreciate your review! :)

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**